plugins/woocommerce-blocks/assets/js/atomic/blocks/product-elements/stock-indicator/attributes.ts

```diff
@@ -16,6 +16,14 @@ export const blockAttributes: BlockAttributes = {
 		type: 'boolean',
 		default: false,
 	},
+	isDescendentOfSingleProductTemplate: {
+		type: 'boolean',
+		default: false,
+	},
+	selectedProductType: {
+		type: 'string',
+		default: 'simple',
+	},
```

Comment:
Do you know if we could make this dynamic so it listens to the store instead of being a block attribute? I know currently the store is closely tighted to the _Add to Cart with Options (Experimental)_ block, but I think we should work on making it more generic so it can end up being used by any other block (like Stock Indicator). What do you think?

Reply:
Done in the commit [82c57c5](https://github.com/woocommerce/woocommerce/pull/53358/commits/82c57c566e23bffa8935a26aedd148f64f195bf8)